src/src_V2/modules/manager/financial/components/payables/control-panel/ControlPanel.jsx

```diff
@@ -0,0 +1,27 @@
+import { Box, Button, InputBase, Paper } from '@mui/material';
+import FilterAltIcon from '@mui/icons-material/FilterAlt';
+import ViewColumnIcon from '@mui/icons-material/ViewColumn';
```

Comment:
i think we can use named import here as we discussed.

Reply:
@igortokhunts jan, unfortunately `@mui/icons-material` doesn't have named export of all these icons, it has only exports of default ones. More about this [here](https://mui.com/components/icons/). You can also look into our `Header.jsx` component of `NotificationsIcon` import.

Reply:
@aninaslyan jan please try FilterAlt instead of FilterAltIcon and ViewColumn instead of ViewColumnIcon.

Reply:
The`NotificationsIcon` is the only component here i think its ok this import style in this case but we can change it as well.